### ISSUE TITLE
想询问您关于该项目的数据集问题

### ISSUE THREAD

**Issue by yueyueniaoyun** [yeyupiaoling/VoiceprintRecognition-Pytorch#2]
笔者您好，最近在学习声纹识别，觉得您的项目和好，但是不明白您create_data.py文件中的def get_data_list(infodata_path, list_path, zhvoice_path):这行代码中的三个变量对应的数据是什么，以及不知道如何编写json文件，请问能否将您项目使用的数据集和json文件按照 get_data_list('dataset/zhvoice/text/infodata.json', 'dataset', 'dataset/zhvoice')这个传入格式发给我方便我理解学习 ，十分感谢您。我的邮箱992597171@qq.com

**Comment by yeyupiaoling** [comment]
你看这个文档：https://github.com/yeyupiaoling/VoiceprintRecognition-Pytorch#%E5%88%9B%E5%BB%BA%E6%95%B0%E6%8D%AE

**Comment by yeyupiaoling** [comment]
是这个数据集：中文语音语料数据集

**Comment by yeyupiaoling** [comment]
`dataset/zhvoice/text/infodata.json`是这个数据的一个文件，你解压就知道了

**Comment by yueyueniaoyun** [comment]
非常感谢您，但是我还有一个问题，为什么路径已经有文件了，但是处理数据时显示系统找不到文件.





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yeyupiaoling/VoiceprintRecognition-Pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月9日(星期四) 晚上7:44
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yeyupiaoling/VoiceprintRecognition-Pytorch] 想询问您关于该项目的数据集问题 (#2)





 
dataset/zhvoice/text/infodata.json是这个数据的一个文件，你解压就知道了
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yeyupiaoling** [comment]
你其他数据解压的文件是在`dataset/zhvoice`路径吗？具体是哪个文件找不着，你看一下路径是否准确

**Comment by yueyueniaoyun** [comment]
因为数据集有点大，我只下载了zhthchs30这个数据集，路径这里显示是正确的，这个路径里也有文件，但是读取不到



------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yeyupiaoling/VoiceprintRecognition-Pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月9日(星期四) 晚上9:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yeyupiaoling/VoiceprintRecognition-Pytorch] 想询问您关于该项目的数据集问题 (#2)





 
你其他数据解压的文件是在dataset/zhvoice路径吗？具体是哪个文件找不着，你看一下路径是否准确
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yeyupiaoling** [comment]
那你看一下数据存放的位置跟体系的路径是否保持一致，要注意相对路径

**Comment by yueyueniaoyun** [comment]
您好，非常感谢您的回复，并且有几个问题想向您请教
1.我在看您代码时发现您设计神经网络时在resnet50v2网络后添加了一层BN，一层Dense以及一层ArcNet，但是我不太明白这样设计的优点，而且在网络上也没有找到ArcNet的相关资料
2.您在什么硬件配置上训练该数据集使用了多长时间呢
以下是您的代码，期待您的回复
def create_model(input_shape):
    # 获取模型
    model = tf.keras.Sequential()
    model.add(ResNet50V2(input_shape=input_shape, include_top=False, weights=None, pooling='max'))
    model.add(BatchNormalization())
    model.add(Dense(units=512, kernel_regularizer=tf.keras.regularizers.l2(5e-4), name='feature_output'))
    model.add(ArcNet(num_classes=args.num_classes))
    return model
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yeyupiaoling/VoiceprintRecognition-Pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月9日(星期四) 晚上9:35
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yeyupiaoling/VoiceprintRecognition-Pytorch] 想询问您关于该项目的数据集问题 (#2)





 
那你看一下数据存放的位置跟体系的路径是否保持一致，要注意相对路径
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yeyupiaoling** [comment]
@yueyueniaoyun  这个网络是参考了ArcFace的，随便改了主体网络结构的，没有特殊设计。
大概训练了5天，双卡

**Comment by yueyueniaoyun** [comment]
非常感谢您，但是我在执行create_data.py处理数据集的时候总是找不到文件路径，您能否将处理为wav格式的部分数据集和test_list.txt文件发送给我，方便我下一步训练，谢谢了




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yeyupiaoling/VoiceprintRecognition-Pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月11日(星期六) 下午2:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yeyupiaoling/VoiceprintRecognition-Pytorch] 想询问您关于该项目的数据集问题 (#2)





 
@yueyueniaoyun  这个网络是参考了ArcFace的，随便改了主体网络结构的，没有特殊设计。
 大概训练了5天，双卡
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yeyupiaoling** [comment]
@yueyueniaoyun 你执行这个程序一般就可以的：https://github.com/yeyupiaoling/VoiceprintRecognition-Pytorch/blob/master/create_data.py

下面是我的list文件：
[list.zip](https://github.com/yeyupiaoling/VoiceprintRecognition-Pytorch/files/7159245/list.zip)

**Comment by yueyueniaoyun** [comment]
您好，我这几天就是用的这个方法试了很多次，但是都无法处理，总是报错路径不对，所以希望您把您处理完的wav格式文件发给我，如果太大只需要整个数据集中的部分即可，谢谢您




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yeyupiaoling/VoiceprintRecognition-Pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月14日(星期二) 下午2:11
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yeyupiaoling/VoiceprintRecognition-Pytorch] 想询问您关于该项目的数据集问题 (#2)





 
@yueyueniaoyun 你执行这个程序一般就可以的：https://github.com/yeyupiaoling/VoiceprintRecognition-Pytorch/blob/master/create_data.py
 
下面是我的list文件：
 list.zip
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by yeyupiaoling** [comment]
额，你下载下载数据集，放在dataset目录下，解压全部压缩包，然后执行python create_data.py就可以了。

**Comment by yueyueniaoyun** [comment]
谢谢您，好像是Linux和windows的路径符号不一样，请问您使用的是哪种操作系统


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "yeyupiaoling/VoiceprintRecognition-Pytorch"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年9月15日(星期三) 下午2:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [yeyupiaoling/VoiceprintRecognition-Pytorch] 想询问您关于该项目的数据集问题 (#2)





 
额，你下载下载数据集，放在dataset目录下，解压全部压缩包，然后执行python create_data.py就可以了。
 
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
